starknetdata/feeder/feeder_test.go

```diff
@@ -206,7 +208,7 @@ func TestTransaction(t *testing.T) {
 		require.NoError(t, err)
 
 		assert.Equal(t, responseTx.Hash, invokeTx.Hash())
-		assert.Equal(t, responseTx.ContractAddress, invokeTx.ContractAddress)
+		assert.Equal(t, responseTx.SenderAddress, invokeTx.ContractAddress)
```

Comment:
Should it be this instead?
```suggestion
		assert.Equal(t, responseTx.ContractAddress, invokeTx.SenderAddress)
```

The 2 values here are nil and I'm not sure if a valid invoke transaction should have a nil sender. We should probably also modify our implementation to reflect same names as it were before. (e.g. `assert.Equal(t, responseTx.ContractAddress, invokeTx.ContractAddress)`).